.betterer.results

```diff
@@ -2768,6 +2768,9 @@ exports[`better eslint`] = {
     "public/app/features/live/index.ts:5381": [
       [0, 0, 0, "Do not use any type assertions.", "0"]
     ],
+    "public/app/features/logs/logsFrame.ts:5381": [
+      [0, 0, 0, "Unexpected any. Specify a different type.", "0"]
+    ],
```

Comment:
I think this is a relict and might need a `yarn betterer --update`?

Reply:
good catch. done!